scripts/default

```diff
@@ -5,4 +5,4 @@ cd $(dirname $0)
 
 ./build
 ./test
-./package
+#./package
```

Comment:
Is there a reason this step is getting skipped?

Reply:
This repo doesn't actually have a `/scripts/package` file currently.